src/VolleyManagement.Backend/VolleyManagement.Contracts/ITeamService.cs

```diff
@@ -53,12 +53,5 @@ public interface ITeamService
         /// <param name="teamId">Id of team which players should be found</param>
         /// <returns>Collection of team's players</returns>
         ICollection<Player> GetTeamRoster(int teamId);
-
-        /// <summary>
-        /// Sets team id to roster
-        /// </summary>
-        /// <param name="roster">Players to set the team id</param>
-        /// <param name="teamId">Id of team which should be set to player</param>
-        void UpdateRosterTeamId(ICollection<Player> roster, int teamId);
```

Comment:
What feature this method

Reply:
Updated previous comment

Reply:
![image](https://user-images.githubusercontent.com/11724508/38022831-46cc6dc0-3289-11e8-81e5-8fa499628c75.png)

Reply:
What do you mean? Update comment of UpdateRosterTeamId?

Reply:
How we can remove teamId from Players and not to decrease functianality?

Reply:
I have strong internal desire to change method, not to erase them. And add value object palayerId)))

Reply:
We should try to do this if it's reasonable. So I'm asking what feature was supported by this method